stdlib/2and3/decimal.pyi

```diff
@@ -1,15 +1,20 @@
+import numbers
 import sys
+from types import TracebackType
 from typing import (
-    Any, Dict, NamedTuple, Optional, Sequence, Tuple, Union, Text,
-    SupportsAbs, SupportsFloat, SupportsInt, SupportsRound,
+    Any, Dict, NamedTuple, Optional, Sequence, Tuple, Union, Text, Type, Container, List,
 )
 
 _Decimal = Union[Decimal, int]
-_ComparableNum = Union[Decimal, int, float]
+_DecimalNew = Union[Decimal, int, float, Text, Tuple[int, Sequence[int], int]]
+if sys.version_info >= (3,):
+    _ComparableNum = Union[Decimal, numbers.Rational, float]
```

Comment:
This conditional definition makes it look confusingly like `int` is not supported in Python 3. In fact, it is redundant in Python 2 too, because it's automatically promoted to `float`. It would be clearer to have `int` either in both or in neither of the branches.